pages/EC-WebLogic_help.xml

```diff
@@ -200,6 +200,10 @@
                         <td>JAVA_HOME environment variable. It should be used when java is not in the path, or different java need to be used.</td>
```

Comment:
AVA_HOME environment variable. It should be used when java is not in the path, or a different JDK needs to be used.